src/algebra/phi-function.md

```diff
@@ -143,6 +143,10 @@ $$a^n \equiv a^{n \bmod \phi(m)} \pmod m$$
 
 This allows computing $x^n \bmod m$ for very big $n$, especially if $n$ is the result of another computation, as it allows to compute $n$ under a modulo.
 
+In group theory, $\phi(n)$ is the [order of the multiplicative group mod n](https://en.wikipedia.org/wiki/Multiplicative_group_of_integers_modulo_n) $(\mathbb Z / n\mathbb Z)^\times$, that is the group of multiplicative units (elements with inverses). The elements with multiplicative inverses are precisely those coprime to $n$.
+
+By Lagrange's Theorem, the multiplicative order of any $a$ must divide $\phi(n)$. If the multiplicative order of $a$ is $\phi(n)$, the largest possible, then $a$ is a [primitive root](primitive-root.md) and the group is cyclic by definition. 
```

Comment:
> of any $a$

What do you mean with $a$? Do you mean every subgroup of $a$, and the order of that subgroup?
Then the first sentence is correct, but the second doesn't make sense.

Or do you mean every element $a$ in that group, and the order of the element?
Because that's not true. E.g. $7 \in \mathbb Z / 8\mathbb Z$, but $\phi(7) \nmid \phi(8)$.
Also the second sentence doesn't make any sense here, right? With a primitiv root you need to generate all elements, so the multiplicative order of $a$ must be $n-1$.

Reply:
I'm not sure if we even need to say this sentence here. I don't think it's that relevant.

Reply:
The [multiplicative order](https://en.wikipedia.org/wiki/Multiplicative_order?wprov=sfla1) of a is the smallest k>0 s.t. a^k = 1 mod n. The group is the multiplicative group, not the group/ring Z/nZ. So 7 is an element of (Z/8Z)*. 7 has order 2 and the multiplicative group has order phi(7) = 4. 
I'm not sure which sentence you don't think is useful because I think it is all useful for math problems.

Reply:
a is an element of the multiplicative group. I should clarify that.

Reply:
Ah, right. Makes sense. My algebra knowledge is not the freshest any more, but it makes sense. The multiplicative order of an element $a$ corresponds to the number of multiplications that it takes to reach 1. Those elements therefor form a group, in particularly even a subgroup. And therefore Lagrange's Theorem applies, and $ord(a) \mid phi(n)$.

Reply:
Yes, $a$ generates a subgroup. I've linked to https://en.wikipedia.org/wiki/Multiplicative_order for more references.